### PR TITLE
refactoring(test): refactored gem integration test

### PR DESCRIPTION
**Pull request by g4s8**
Extracted `setUp` and `tearDown` methods to extend test later.
Removed redundant temporary directory mounting, copy data to
container directly.
Removed empty log messages.

Ticket: #97

### CODE REVIEW

File: src/test/java/com/artipie/gem/GemCliITCase.java [hunk 1]
```diff
@@ -36,70 +35,100 @@
...
+            .withCommand("tail", "-f", "/dev/null");
+        Testcontainers.exposeHostPorts(port);
+        this.container.start();
+    }
+
+    @AfterEach
+    void tearDown() throws Exception {
+        if (this.server != null) {
+            this.server.stop();
+            this.server.close();
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/gem/GemCliITCase.java:80]: As far as I remember, `close()` calls `stop()`, so there is no need to call both methods

File: src/test/java/com/artipie/gem/GemCliITCase.java [hunk 2]
```diff
@@ -36,70 +35,100 @@
...
+        }
+        if (this.vertx != null) {
+            this.vertx.close();
+        }
+        if (this.container != null) {
+            this.container.stop();
+            this.container.close();
+        }
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/gem/GemCliITCase.java:88]: Same here

File: src/test/java/com/artipie/gem/GemCliITCase.java [hunk 3]
```diff
@@ -36,70 +35,100 @@
...
-        gems.add("file-tail-1.2.0.gem");
         for (final String gem : gems) {
-            final Path target = mount.resolve(gem);
-            try (InputStream is = this.getClass().getResourceAsStream("/".concat(gem));
-                OutputStream os = Files.newOutputStream(target)) {
-                IOUtils.copy(is, os);
-            }
+            this.container.copyFileToContainer(
+                Transferable.of(
+                    IOUtils.toByteArray(this.getClass().getResourceAsStream("/".concat(gem)))
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/gem/GemCliITCase.java:103]: Is not it possible to use `TestResourse` from asto and simplify the expression?

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@g4s8 please check my comments

**Comment by g4s8** [comment]
@olenagerasimova thanks, fixed

**Review by olenagerasimova — APPROVED** [review]
@g4s8 thanks

**Pull request merged**